src/openforms/authentication/models.py

```diff
@@ -347,6 +349,21 @@ def to_auth_context_data(
                         "branchNumber"
                     ] = branch_number
                 return ehm_context
+
+            # Experimental feature that is different from the defined authentication context.
+            # This will have to be re-worked and re-thought-out, and will most definitely change.
+            # This is a fix for #4863
+            case (AuthAttribute.employee_id, ""):
+                employee_context: EmployeeContext = {
+                    "source": "custom",
+                    "levelOfAssurance": self.loa,
+                    "representee": {
+                        "identifierType": "employee_id",
+                        "identifier": self.value,
+                    },
+                }
+                return employee_context
```

Comment:
Some background to explain more about how things work in Open Forms!

The model `AuthInfo` (which defines the `to_auth_context_data` method) captures who is authenticated for the submission. This means that if an employee of the organization (after OIDC login) logs in for the form for themselves (the other flow I will describe below), they are the `authorizee.legalSubject` (representing themselves). This is a rather simple setup of employee context that needs to be returned if we follow the established pattern (where representee is implied from the authorizee/legal subject).

```json
{
  "source": "custom",
  "loa": "unknown",
  "authorizee": {
    "legalSubject": {
      "identifierType": "opaque",
      "identifier": self.value
    }
  }
}
```

Identifier type opaque means you can't derive any meaning/format validation from it.

The other flow is where an employee is logged in with OIDC, and they enter the BSN or KVK of the customer. In the code, this brings you to `openforms.authentication.signals.set_auth_attribute_on_session` in the branch with `if registrator_subject`. In that situation, on the `AuthInfo` instance not the employee details are stored, but the BSN/KVK nr of the actual customer, and the `attribute` is then set to `bsn` or `kvk` rather than `employee_id`, so you can never end up in this case in those flows. Meaning, the `to_auth_context_data` only needs to consider the situation where an employee is acting on their own behalf which is nice because it simplifies things a bit.

Looking at the Sentry error, this is the right place to patch this up.

So, key takeaways:

* the context dict/shape here needs to use `authorizee > legalSubject` structure, similar to DigiD without machtigen (first case)
* we don't need to consider "acting on behalf of someone else" for employee_id attribute

Reply:
Okay.. I think i'm starting to understand this a bit more..

Should the context then not also contain the `representee` data? (Although this would probably be the same information.., right?)

Reply:
if no representee is specified, it is indeed (implicitly) equal to `authorizee.legalSubject`